clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/client/model/Job.java

```diff
@@ -148,6 +156,68 @@ public Job(com.netflix.titus.grpc.protogen.Job grpcJob, List<com.netflix.titus.g
 
   }
 
+  private void addDisruptionBudget(com.netflix.titus.grpc.protogen.Job grpcJob) {
```

Comment:
you should do an import for this so it's just `Job grpcJob`. Intellij should show you that ("add static import") if you option click on it

Reply:
Unfortunately, this class is also named `Job`, so we have to fully qualify this one.

Reply:
Oof. Sorry. Such great code 👍